specification/sql/resource-manager/Microsoft.Sql/preview/2017-10-01-preview/shortTermRetentionPolicies.json

```diff
@@ -219,12 +264,44 @@
         }
       }
     },
+    "Resource": {

```

Comment:
Please delete these and manually update to use the common definitions, e.g. "../../../common/v1/types.json#/definitions/ProxyResource"

Reply:
Done. Thanks!